rpmlint/checks/RCLinksCheck.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+import stat
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class RCLinksCheck(AbstractCheck):
+    def check(self, pkg):
+        if pkg.isSource():
+            return
+
+        rclinks = set()
+        rccandidates = set()
+        initscripts = set()
+
+        for fname, pkgfile in pkg.files().items():
+            if fname in pkg.ghostFiles():
+                continue
+
+            if (stat.S_ISLNK(pkgfile.mode) and
+                    (fname.startswith('/usr/sbin/rc') or
+                     fname.startswith('/sbin/rc'))):
+                rclinks.add(fname.partition('/rc')[2])
+            elif fname.startswith('/usr/lib/systemd/system/'):
+                basename = Path(fname).name
+                if '@' in fname:
+                    continue
+                if fname.endswith('.service') or fname.endswith('.target'):
+                    rccandidates.add(basename.rpartition('.')[0])
+            elif fname.startswith('/etc/init.d/'):
+                basename = Path(fname).name
+                if not (basename.startswith('rc') or
+                        basename.startswith('boot.')):
+                    initscripts.add(basename)
+
+        for fname in sorted(initscripts):
+            if fname in rccandidates:
+                self.output.add_info('E', pkg, 'systemd-shadowed-initscript', fname)
```

Comment:
This should for sure go to upstream at:
  https://github.com/rpm-software-management/rpmlint/blob/master/rpmlint/checks/SysVInitOnSystemdCheck.py

Reply:
Didn't I say this in the original rclinks PR?

Reply:
If so, then the 2 checks should be probably merged. So that I'm closing the PR for now.